projects/akita-ng-fire/src/lib/collection/collection.service.ts

```diff
@@ -350,7 +358,8 @@ export class CollectionService<S extends EntityState<EntityType, string>, Entity
       setLoading(storeName, true);
     }
     return this.db.doc<EntityType>(path).valueChanges().pipe(
-      map((entity) => {
+      map(value => this.formatFromFirestore(value)),
```

Comment:
I would recommend to do that at line 367 instead (after checking if entity exist) else you would need to do the check on every `formatFromFirestore` : 
```typescript
const data = this.formatFromFirestore({ [this.idKey]: id, ...entity });
```